js/components/interface/listViewer/ListViewer.js

```diff
@@ -0,0 +1,340 @@
+import React from 'react';
+import Griddle, { plugins, ColumnDefinition, RowDefinition } from 'griddle-react';
+import BaseIconComponent from './BaseIconComponent';
+
+import PopupColorPicker from './PopupColorPicker';
+import './listviewer.less';
+
+
+function mapToObject ( aMap ) {
```

Comment:
Inconsistent spacing between these utility functions or is the linter somehow doing that with some rules?

Also, I think we had a place with stuff like this already, worth considering moving this garbage (map to object and isString at least?) there --> https://github.com/openworm/geppetto-client/blob/development/js/common/GEPPETTO.Utility.js

They're likely to be re-used all over the place and in general I don't like random functions in a module out of class scope 😬

Reply:
Yes I'm using eslint for formatting. Maybe there is no enforcement rule for function spacing, as there is no rule for semi colons.
I'll move the functions to utility as suggested

Reply:
I've been looking at GEPPETTO.Utility.js: it's an old prototype requiring a GEPPETTO instance and other libraries included jquery. I prefer moving the utility functions in another file inside the component if it's ok for you

Reply:
Yeah doesn't matter where, that utility stuff is pretty old and utility buckets suck in general 😂if you have a better alternative to group that functionality in some way that makes semantic sense it can only be good